packages/panw_cortex_xdr/data_stream/alerts/manifest.yml

```
@@ -110,5 +118,123 @@ streams:
           Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/beats/filebeat/current/filtering-and-enhancing-data.html) for details.
 
     template_path: httpjson.yml.hbs
-    title: Palo Alto Cortex XDR Alerts
-    description: Palo Alto Cortex XDR Alerts API
+    title: Palo Alto Cortex XDR Alerts from v1 API - Deprecated
+    description: Palo Alto Cortex XDR Alerts from v1 API. This API is deprecated. Please use v2 API using CEL input instead.
```

I don't think we need to mention CEL here.
```suggestion
    description: Palo Alto Cortex XDR Alerts from v1 API. This API is deprecated. Please use v2 API instead.
```

---

Updated as per suggestion in https://github.com/elastic/integrations/commit/74f31e3cb64adc8a54baabc8e708daea73fc45d1